pynxtools/dataconverter/helpers.py

```diff
@@ -446,7 +447,9 @@ def does_group_exist(path_to_group, data):
     return False
 
 
-def ensure_all_required_fields_exist(template, data, nxdl_root):
+# pylint: disable=W1203
+def ensure_all_required_fields_exist(template, data,
+                                     nxdl_root, logger=pynx_logger):
```

Comment:
Normally, it shouldn't be necessary to pass the logger here. `logging` keeps track of the available loggers and integrates them